### PR TITLE
Remove height from shape metric

### PR DESCRIPTION
**Pull request by ronald-jaepel**
solves #104

### CODE REVIEW

File: CADETProcess/comparison/difference.py [hunk 1]
```diff
@@ -466,6 +466,7 @@ class Shape(DifferenceBase):
     def __init__(
             self, *args,
             use_derivative=True, normalize_metrics=True, normalization_factor=None,
+            include_height=False,
```
Comment (schmoelder) [inline CADETProcess/comparison/difference.py:469]: Please udpate the docstring

File: CADETProcess/comparison/difference.py [hunk 2]
```diff
@@ -535,14 +539,21 @@ def __init__(
 
     @property
     def n_metrics(self):
+        n_metrics = 2
+
+        if self.include_height:
+            n_metrics += 1
+
         if self.use_derivative:
```
Comment (schmoelder) [inline CADETProcess/comparison/difference.py:547]: Should we also only optionally compute peak height difference for the derivatives (i.e. if self.include_height)?

**Comment by schmoelder** [comment]
> solves #104

You can also link to Issues to close them automatically when adding them in the development panel on the right:

grafik

File: CADETProcess/comparison/difference.py (hunk 2 above)
Comment (ronald-jaepel) [reply CADETProcess/comparison/difference.py:551]: I've changed it to do that.
```
        include_height : bool, optional
            If True, also return a metric for the height difference.
            If `use_derivative` is also true, also return height differences
            for the derivatives.
```

**Comment by ronald-jaepel** [comment]
Done with requested changes. Ready to merge imo.

**Pull request merged**